azure-pipelines.yml

```diff
@@ -453,8 +477,47 @@ stages:
             inputs:
               artifactName: conda_default
 
-          - task: GitHubRelease@1
+          - script: |
```

Comment:
As far as I remember script simply defaults to the build agent's (here Ubuntu Linux) shell, so it would also be bash. I simply copied another step to define this one.